client/tsconfig.json

```diff
@@ -16,7 +16,7 @@
     "sourceMap": true,
     "strict": true,
     "target": "es2020",
-    "baseUrl": "src"
+    "baseUrl": "./src"
```

Comment:
Could you explain more about this changes?

Reply:
I was trying to run the expo simulator through the iPhone (in a Windows PC), but an error occurred, so I tried this.
I asked this issue on Slack at the time(August 18), and I modified it with a relative path by referring to the comments posted there.

Reply:
Thanks for the reply. I am unsure though since this isn't happening in my environment. Could you kindly verify this once again and then reply back? If this happens intermittently, I'd love to apply these changes either.

Reply:
I tested it several times with the source changed, but I couldn't fix the error. So I think that tsconfig.js changes(src to ./src) need not to be applied.

Reply:
I tested it several times with the source changed, but I couldn't fix the error. So I think that tsconfig.js changes(src to ./src) need not to be applied. I should have changed this to the original, but I didn't change it by mistake😥.